crates/symbolicator-sources/src/index.rs

```diff
@@ -0,0 +1,104 @@
+use std::collections::BTreeSet;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader, Write};
+use std::path::Path;
+use std::sync::Arc;
+
+#[derive(Debug, Clone, Default)]
+pub struct SymstoreIndex {
+    files: BTreeSet<Arc<str>>,
+}
+
+impl SymstoreIndex {
+    fn parse_line(line: &str) -> Option<Arc<str>> {
+        let entry = line.split('"').nth(1)?;
+        let (name, id) = entry.split_once('\\')?;
+        Some(format!("{name}/{id}/{name}").into())
+    }
```

Comment:
A line is always
```
"<name>\<id>",<some other stuff>.
```
This turns such a line into
```
<name>/<id>/<name>,
```
which is what the path actually looks like on the server.

Reply:
Should put this into the code.